### PR TITLE
Small improvements to Fabric Video Expandable v2

### PR DESCRIPTION
**Pull request by regiskuckaertz**
Some ch-ch-ch-ch-changes:
- The ad collapses when the user clicks the CTA
- The plus icon appears all the time, even when in the container and the ad is collapsed (don't ask)
- The cursor turns into a pointer when the ad is collapsed (the whole area is clickable)
- Made sure it works fine in iOS, IE10 and IE11
- Made sure the video is full width in full-width mode

### CODE REVIEW

**Comment by regiskuckaertz** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;dzjllgzi&#x2F;141-fabric-video-expandable-v2-changes-and-fixes"> Fabric Video Expandable v2: changes and fixes</a>

File: static/src/javascripts/projects/commercial/modules/creatives/fabric-expandable-video-v2.js [hunk 1]
```diff
@@ -32,17 +32,15 @@ define([
 
         function create() {
             var videoHeight = openedHeight;
+            var plusIconPosition = params.showCrossInContainer.substring(3);
```
Comment (piuccio) [inline static/src/javascripts/projects/commercial/modules/creatives/fabric-expandable-video-v2.js]: what's the magic number `3`?

File: static/src/javascripts/projects/commercial/modules/creatives/fabric-expandable-video-v2.js [hunk 2]
```diff
@@ -99,6 +70,44 @@ define([
...
+                    $('.ad-exp__close-button', $adSlot[0]).removeClass('button-spin');
+                    $('.ad-exp__open-chevron', $adSlot[0]).removeClass('chevron-down');
+                    $ad.css('height', closedHeight);
+                    $fabricExpandableVideo.removeClass('creative--open');
+                    $('.slide-video, .slide-video .ad-exp__layer', $adSlot[0])
+                    .css('height', closedHeight)
+                    .removeClass('slide-video__expand');
+                }
+
+                isClosed = !isClosed_;
```
Comment (piuccio) [inline static/src/javascripts/projects/commercial/modules/creatives/fabric-expandable-video-v2.js]: kinda confusing names for these two variables. One is a state, the other a transition

**Review by piuccio — APPROVED** [review]
minor things

**Comment by rich-nguyen** [comment]
👍  apart from @piuccio 's comments

File: static/src/javascripts/projects/commercial/modules/creatives/fabric-expandable-video-v2.js (hunk 1 above)
Comment (regiskuckaertz) [inline static/src/javascripts/projects/commercial/modules/creatives/fabric-expandable-video-v2.js]: some stupid convention in the DFP template, strings will come as `To something` and I need the `something` part.

**Pull request merged**